### PR TITLE
Creating the database

### PR DESCRIPTION
**Pull request by Rayane0823**

### CODE REVIEW

**Review by LucasMerino10 — APPROVED** [review]

**Review by Romlion — APPROVED** [review]

File: backend/database/schema.sql [hunk 1]
```diff
@@ -1,4 +1,74 @@
...
+  title varchar(80) not null,
+  technique varchar(50) not null,
+  artwork_year int not null,
+  format varchar(50) not null,
+  source varchar(150) not null,
+  alt varchar(80) not null,
+  user_id_ar int not null,
+  constraint fk_artwork_artist
+  foreign key (user_id_ar)
+  references artist(user_id)
```
Comment (Yoann-Fortin) [inline backend/database/schema.sql]: user_id isn't a primary key

File: backend/database/schema.sql [hunk 2]
```diff
@@ -1,4 +1,74 @@
...
+);
+create table management (
+  user_id_ad int not null,
```
Comment (Yoann-Fortin) [inline backend/database/schema.sql:48]: id is missing

File: backend/database/schema.sql [hunk 3]
```diff
@@ -1,4 +1,74 @@
...
+  constraint fk_management_administrator
+  foreign key (user_id_ad)
+  references administrator(user_id),
```
Comment (Yoann-Fortin) [inline backend/database/schema.sql]: user_is isn't a primary key

File: backend/database/schema.sql [hunk 4]
```diff
@@ -1,4 +1,74 @@
...
+  artwork_id int not null,
+  constraint fk_management_artwork
+  foreign key (artwork_id)
+  references artwork(id),
+  uploading_date date not null,
+  approving_date date not null,
+  modification_date date not null
+);
+create table favorite (
+  user_id_lu int not null,
```
Comment (Yoann-Fortin) [inline backend/database/schema.sql:62]: id is missing

File: backend/database/schema.sql [hunk 5]
```diff
@@ -1,4 +1,74 @@
...
+  constraint fk_favorite_logged_user
+  foreign key (user_id_lu)
+  references logged_user(user_id),
```
Comment (Yoann-Fortin) [inline backend/database/schema.sql]: user_id isn't a primary key

File: backend/database/schema.sql [hunk 6]
```diff
@@ -1,4 +1,74 @@
...
+  constraint fk_management_artwork
+  foreign key (artwork_id)
+  references artwork(id),
+  uploading_date date not null,
+  approving_date date not null,
+  modification_date date not null
+);
+create table favorite (
+  user_id_lu int not null,
+  constraint fk_favorite_logged_user
```
Comment (Yoann-Fortin) [inline backend/database/schema.sql:63]: An artist or an admin can't have a favorite picture ?

File: backend/database/schema.sql [hunk 7]
```diff
@@ -1,4 +1,74 @@
...
+  foreign key (user_id_lu)
+  references logged_user(user_id),
+  artwork_id int not null,
+  constraint fk_favorite_artwork
+  foreign key (artwork_id)
+  references artwork(id),
+  liked boolean not null
```
Comment (Yoann-Fortin) [inline backend/database/schema.sql]: Why this field ?

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: backend/database/schema.sql (hunk 2 above)
Comment (Rayane0823) [reply backend/database/schema.sql:48]: There is a many-to-many relation between admin and artwork tables. Management doesn't need an id, it's an intermediary table

File: backend/database/schema.sql (hunk 4 above)
Comment (Rayane0823) [reply backend/database/schema.sql:62]: There is a many-to-many relation between logged_user and artwork tables. Favorite doesn't need an id, it's an intermediary table

**Review by pagouillart — APPROVED** [review]

**Review by Romlion — APPROVED** [review]

**Review by Cyrildab — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**